localgov_eu_cookie_compliance.module

```diff
@@ -116,6 +116,35 @@ function localgov_eu_cookie_compliance_form_eu_cookie_compliance_config_form_alt
     '#convert_path' => PathElement::CONVERT_NONE,
   ];
 
+  // Insert the oembed management section.
+  // First, we define the section.
+  $section = [
+    '#type' => 'details',
+    '#title' => t("Embed handling"),
+    '#open' => TRUE,
+    '#states' => [
+      'visible' => [
+        "input[name='method']" => ['!value' => 'default'],
+      ],
+    ],
+  ];
+
+  // Then find where in the associative array the cookies section is, and
+  // splice the array so the new section is immediately after it in the
+  // implicit ordering. This prevents the new section being at the bottom
+  // of the form, and instead places it with logical siblings.
+  $offset = array_search("cookies", array_keys($form)) + 1;
+  $form = array_slice($form, 0, $offset, TRUE) +
+          ['oembed' => $section] +
+          array_slice($form, $offset, NULL, TRUE);
+
+  $form['oembed']['disabled_oembeds'] = [
+    '#type' => 'textarea',
+    '#title' => t('Disable OEmbed sources'),
+    '#default_value' => $config->get('disabled_oembeds'),
```

Comment:
Please declare `disabled_oembed` in the [schema file](https://github.com/localgovdrupal/localgov_eu_cookie_compliance/blob/1.x/config/schema/localgov_eu_cookie_compliance.schema.yml).

Reply:
See, this is the kind of thing I wanted the Nomensa PHP devs to check it for first :D Thanks!